src/azure-cli/azure/cli/command_modules/acs/commands.py

```diff
@@ -24,14 +24,16 @@
 def load_command_table(self, _):
 
     container_services_sdk = CliCommandType(
```

Comment:
Please add resource type information in CliCommandType.

Reply:
@hhz1992 to address this